cmd/skupper/skupper_cluster_test.go

```diff
@@ -1595,3 +1609,71 @@ func TestNetworkStatusWithCluster(t *testing.T) {
 		testCommand(t, cmd, tc.doc, tc.expectedError, tc.expectedCapture, tc.expectedOutput, tc.outputRegExp, tc.args...)
 	}
 }
+
+func TestLinkStatus(t *testing.T) {
+	testcases := []testCase{
+		{
+			doc:             "Should not display any detail if the link does not exist",
+			args:            []string{"link1", "--verbose"},
+			expectedCapture: "No such link",
+			expectedOutput:  "",
+			expectedError:   "",
+			realCluster:     true,
+			createConn:      false,
+		},
+		{
+			doc:             "Should display local links",
+			args:            []string{},
+			expectedCapture: "Links initiated from this site",
+			expectedOutput:  "",
+			expectedError:   "",
+			outputRegExp:    "",
+			realCluster:     true,
+			createConn:      true,
+		},
+		{
+			doc:             "Should display remote links",
+			args:            []string{},
+			expectedCapture: "Currently active links from other sites",
+			expectedOutput:  "",
+			expectedError:   "",
+			outputRegExp:    "",
+			realCluster:     true,
+			createConn:      true,
+		},
+	}
+
+	namespace = "cmd-link-cluster-test-" + strings.ToLower(utils.RandomId(4))
+	kubeContext = ""
+	kubeConfigPath = ""
+
```

Comment:
It might be better to call NewClient here, as running this test individually is causing it to fail.

Reply:
Done! ✅